### DISCUSSION TITLE
error running example code

### DISCUSSION THREAD

**Discussion by Xinling-Guo** [alibaba/GraphScope#2278] [Q&A]
Hi, I was trying to play with GraphScope from its example code. and when I was trying to load the graph with interactive = graphscope.gremlin(g), the bug will happen saying "[IOError](cat: /tmp/gs/xnlzrm/session_tkezkmjp/pid/23417489089432882/frontend.pid: No such file or directory)" (see the attached screenshot for detail).

![bug](https://user-images.githubusercontent.com/57967707/205628847-795532a4-90b2-4ae1-b5b5-febf3fc0426d.png)

**Comment by Taotiee** [comment]
I have the same problem. How did you solve it?
